src/components/Details/VmTemplateDetails/VmTemplateDetails.js

```diff
@@ -124,10 +142,10 @@ export class VmTemplateDetails extends React.Component {
   isFormValid = () => Object.keys(this.state.form).every(key => this.state.form[key].valid);
 
   render() {
-    const { vmTemplate, dataVolumes, NamespaceResourceLink, LoadingComponent, k8sGet } = this.props;
+    const { vmTemplate, dataVolumes, NamespaceResourceLink, LoadingComponent } = this.props;
     const id = getId(vmTemplate);
     const vm = selectVm(vmTemplate.objects);
-    const baseTemplate = getVmTemplate(vmTemplate);
+    const baseTemplateLabels = getVmTemplate(vmTemplate);
```

Comment:
please can you rename this method to something like getVmTemplateLabels/getVmTemplateInfo. Or even better rename it to getTemplateDisplayName and reimplement it so that it returns string?

Reply:
sure :smile_cat: